app.rb

```diff
@@ -0,0 +1,112 @@
+require './book'
+require './person'
+require './rental'
+require './teacher'
+require './student'
+
+class App
+  def on_start
+    puts '--------------------------------------------------'
+    puts 'Select an option: '
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals'
+    puts '7 - Exit'
+    puts '--------------------------------------------------'
+  end
+
+  def list_all_books
+    puts 'There\'s no books added yet!' if Book.all.empty?
+    Book.all.each do |book|
+      puts %(Title: #{book.title}, Author: #{book.author})
+    end
+  end
+
+  def list_all_people
+    puts 'There\'s no people added yet' if Student.all.empty? && Teacher.all.empty?
+    Student.all.each do |student|
+      puts %([Student] Name: #{student.name}, ID: #{student.id}, Age: #{student.age})
+    end
+    Teacher.all.each do |teacher|
+      puts %([Teacher] Name: #{teacher.name}, ID: #{teacher.id}, Age: #{teacher.age})
+    end
+  end
```

Comment:
The app abruptly closes when there is no book or people availabe. This is not supposed to be so, it should allow you to create a book or person and also check again to create rentals. Kindly check and fix this issue. 

![Screenshot from 2023-01-06 23-23-31](https://user-images.githubusercontent.com/65233860/211115959-9faf9a8d-2281-4101-b069-b7888252797b.png)


![Screenshot from 2023-01-06 23-23-22](https://user-images.githubusercontent.com/65233860/211115950-c9660778-b3cc-4738-9821-8e7f3d6d2b79.png)